### PR TITLE
Feat/login page redesign

### PR DESCRIPTION
**Pull request by HatemMn**
This PR goal is to change the landing login page to fit the new design of cosmian.com

### CODE REVIEW

File: src/component/layout.less [hunk 1]
```diff
@@ -13,57 +25,118 @@ ul {
...
+      button {
+        cursor: pointer;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        --border-radius: 44px;
+        border-radius: var(--border-radius);
+        width: 200px;
+        height: 78px;
```
Comment (ccorsin) [inline src/component/layout.less:101]: please reduce this size again - we said around 150px for the width to get the same proportion as the Figma (and adjust height too of course)

File: src/assets/cosmian_logo.svg
Comment (ccorsin) [inline src/assets/cosmian_logo.svg:1]: please adjust the logo of the interactive tutorial with the new one.

File: src/pages/LoginPage.tsx [hunk 2]
```diff
@@ -1,20 +1,60 @@
...
+            <button onClick={signUpWithRedirect}>
+              SIGN UP <ButtonsIcon className="arrow" />
+            </button>
+          </Col>
+          <Col offset={1}>
+            <button onClick={loginWithRedirect}>
+              LOG IN <ButtonsIcon className="arrow" />
+            </button>
+          </Col>
+        </Row>
```
Comment (ccorsin) [inline src/pages/LoginPage.tsx:35]: I don't think we need thos antd elements - as we are not building a table.
Please use simple h1 and div tags, and write a simple CSS with flexbox.
If you need help with flexbox, I'll come.

File: src/assets/cosmian_logo.svg
Comment (HatemMn) [reply src/assets/cosmian_logo.svg:1]: for info, the previous code used the cosmian_ui package
```
import { Button, CosmianLogo, Header } from "cosmian_ui";

            <CosmianLogo />
```
I replaced it with a simple image unless we plan to update cosmian_ui
`            Cosmian logo white
`

File: src/component/layout.less (hunk 1 above)
Comment (HatemMn) [reply src/component/layout.less:101]: image

I ended up chosing 180px && 70px - this is how it looks like on my laptop

File: src/pages/LoginPage.tsx (hunk 2 above)
Comment (HatemMn) [reply src/pages/LoginPage.tsx:35]: I changed it with flexbox & used the gap property to don't lte the elements touche

File: src/assets/cosmian_logo.svg
Comment (HatemMn) [reply src/assets/cosmian_logo.svg:1]: other info : thr fonts are now under assets/fonts

**Review by ccorsin — APPROVED** [review]

**Pull request merged**